pages/12.Plugins/20.Zapier/docs.en.md

```diff
@@ -17,21 +17,30 @@ This integration integrates any Mautic instance with 1.100+ third party web serv
 
 ## Requirements
 
-1. Mautic version 2.9.0 or newer with SSL - basic authentication requires HTTPS to be secure.
-3. Zapier account. Free or Paid. [View Zapier pricing][zapier-pricing].
+Zapier account. Free or Paid. [View Zapier pricing](https://zapier.com/pricing/).
```

Comment:
Please use a link macro - see style guide: https://docs.mautic.org/en/home/contributing#links. The macro already exists at the bottom of the page so we should retain this.

```suggestion
Zapier account. Free or Paid. [View Zapier pricing][zapier-pricing].
```